demos/payment-provider/README.md

```diff
@@ -0,0 +1,29 @@
+# Payment Provider
+
+This demo shows how to make an app that contains the Payment Provider supplied in the `playbilling` module in the Android Browser Helper.
```

Comment:
nit: I've been trying to limit markdown lines to 100 chars

Reply:
Done